assets/js/common/ActivityLogsConfig/ActivityLogsConfig.test.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
+
+import ActivityLogsConfig from '.';
+
+describe('ActivityLogsConfig', () => {
+  it.each`
+    time                           | expected
+    ${'1 day'}                     | ${'1 day'}
```

Comment:
I am not sure in which scenario we would get a plain string rather than the structured value/unit, but it does not harm.